src/pages/VideoPlayer/VideoPlayer.jsx

```diff
@@ -21,20 +28,92 @@ const Container = styled(Grid)({
 });
 function VideoPlayer() {
     const location = useLocation();
+    const history = useHistory();
```

Comment:
this file is starting to get big, the logic and components could be moved to other files